apps/preview/nuxt/pages/showcases/Search/SearchBasic.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <form @submit.prevent="submit" ref="referenceRef" class="relative">
```

Comment:
suggestion: `role="search"` is missing

Reply:
Done [`2e943fa`](https://github.com/vuestorefront/storefront-ui/pull/2630/commits/2e943fa5937d9562836d3ec710b01e4936e1885b)